src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -67,10 +68,7 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
         this.signInLocal.type = TypeSignIn.oauth
         _route.queryParams.subscribe((params) => {
           this.signInLocal.params = {
-            client_id: params['client_id'],
-            response_type: params['response_type'],
-            scope: params['scope'],
-            redirect_uri: params['redirect_uri'],
+            ...(params as OauthParameters),
```

Comment:
This PR is mostly sending every parameter so no other parameters like `state` or `promp` is filter out

Reply:
@DanielPalafox @amontenegro This is one of those confusing PRs that include many commits of another PR #331 

So this is a nice quick way to review only the recent commits only

![image](https://user-images.githubusercontent.com/2119626/87586641-0d63d400-c69e-11ea-9d26-b4128597b3a5.png)


You can use shift and select only the commits after f9cc475 (including it)